packages/@react-spectrum/combobox/stories/ComboBox.stories.tsx

```diff
@@ -414,6 +414,18 @@ storiesOf('ComboBox', module)
       <ControlledKeyComboBox defaultInputValue="Blah" selectedKey="2" disabledKeys={['2', '6']} />
     )
   )
+  .add(
+    'inputValue and isOpen (controlled)',
+    () => (
+      <ControlledValueOpenCombobox />
+    )
+  )
+  .add(
+    'selectedKey and isOpen (controlled)',
+    () => (
+      <ControlledKeyOpenCombobox />
+    )
+  )
```

Comment:
not entirely convinced that these are a combination that users would make, but good test cases for useComboBoxState logic